common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -427,6 +427,11 @@ public class BrokerConfig extends BrokerIdentity {
     // if false, will still rewrite ck after max times 17
     private boolean skipWhenCKRePutReachMaxTimes = false;
 
+    /**
+     * Use AdaptiveBackOffLock
+     **/
+    private boolean useABSLock = false;
```

Comment:
It would be better to put this config to useABSLock

Reply:
done~